src/corelibs/U2Formats/src/tasks/BgzipTask.cpp

```diff
@@ -122,7 +122,12 @@ Task::ReportResult BgzipTask::report() {
 
 bool BgzipTask::checkBgzf(const GUrl& fileUrl) {
     NP<FILE> file = FileAndDirectoryUtils::openFile(fileUrl.getURLString(), "r");
-    int checkResult = file == nullptr ? -1 : bgzf_check_bgzf(file);
+    int checkResult = 0;
```

Comment:
Here you replaced 1 line of code with 5. Does it change anything? If not, please keep the old code